README.md

```diff
@@ -226,13 +226,15 @@ necessary:
    `windows`.
 3. map the container's port on localhost 4000 to the hosts port 4000
 
-With our `dev_device` from above, the call would then look, for example, as follows:
+With our `dev_device` from above, the call on a Windows host would then look,
+for example, as follows:
 
 ```sh
 docker run --rm \
   -u 0:0 \
   -v "~/.ssh":/root/.local/share/omnect-cli \
```

Comment:
in my test setup on windows if must be used the absolute path of the local .ssh folder

Reply:
fixed